src/renderer/pages/Operations/components/ActionSteps/Confirmation.tsx

```diff
@@ -1,22 +1,51 @@
-import { MultisigTransaction, Transaction, Fee } from '@renderer/entities/transaction';
+import { useEffect, useState } from 'react';
+import { useStore } from 'effector-react';
+
+import {
+  MultisigTransaction,
+  Transaction,
+  Fee,
+  DepositWithLabel,
+  isXcmTransaction,
+  TransactionType,
+  XcmFee,
+} from '@renderer/entities/transaction';
 import { TransactionAmount } from '@renderer/pages/Operations/components/TransactionAmount';
-import { DetailRow, FootnoteText, Icon } from '@renderer/shared/ui';
+import { Button, DetailRow, FootnoteText, Icon } from '@renderer/shared/ui';
 import { ExtendedChain } from '@renderer/entities/network';
 import { useI18n } from '@renderer/app/providers';
 import { getIconName } from '../../common/utils';
-import type { MultisigAccount } from '@renderer/shared/core';
+import type { Account, MultisigAccount } from '@renderer/shared/core';
 import Details from '../Details';
+import { sendAssetModel } from '@renderer/widgets';
+import { getAssetById } from '@renderer/shared/lib/utils';
 
 type Props = {
   tx: MultisigTransaction;
   account: MultisigAccount;
+  signatory?: Account;
   connection: ExtendedChain;
   feeTx?: Transaction;
+  onSign: () => void;
 };
-export const Confirmation = ({ tx, account, connection, feeTx }: Props) => {
+export const Confirmation = ({ tx, account, connection, signatory, feeTx, onSign }: Props) => {
+  const xcmConfig = useStore(sendAssetModel.$finalConfig);
```

Comment:
Better to use `useUnit` and move it 2 lines below
`const { t } = useI18n();` is a number 1 line in each component including translation 😉
```suggestion
  const xcmConfig = useUnit(sendAssetModel.$finalConfig);
```